build: Cleans the TensorRTLLM + Dynamo container build (#968)
ai-dynamo/dynamo: examples/tensorrt_llm/README.md

### Build docker

```bash
# TensorRT-LLM uses git-lfs, which needs to be installed in advance.
apt-get update && apt-get -y install git git-lfs

# On an x86 machine:
./container/build.sh --framework tensorrtllm

# On an ARM machine:
./container/build.sh --framework tensorrtllm --platform linux/arm64
```

> [!NOTE]

> > Because of the known issue of C++11 ABI compatibility within the NGC pytorch container,
> > we rebuild TensorRT-LLM from source. See [here](https://nvidia.github.io/TensorRT-LLM/installation/linux.html)
> > for more informantion.
> >
> > Hence, when running this script for the first time. the time taken by this script can be
> > quite long. This is because we are building TensorRT-LLM library from source.

Reviewer: ```suggestion
> Because of a known issue of C++11 ABI compatibility within the NGC pytorch container,
> we rebuild TensorRT-LLM from source. See [here](https://nvidia.github.io/TensorRT-LLM/installation/linux.html)
> for more informantion.
>
> Hence, when running this script for the first time, the time taken by this script can be
> quite long.
```
